seed/views/main.py

```diff
@@ -2460,7 +2537,7 @@ def retrieve_finer_timeseries_data(request):
             res['reading'].append(ts_json)
 
             count = count + 1
-            if count == 10:
+            if not start_time and not end_time and count == 10:
```

Comment:
Sure. Basically at most 10 finer timeseries data samples are returned to
show the data are there. It could be too long if display them all on the
web page.

On Tue, Mar 29, 2016 at 1:01 PM, Nicholas Long <notifications@github.com>
wrote:

> In seed/views/main.py
> <https://github.com/SEED-platform/seed/pull/863#discussion_r57759601>:
>
> > @@ -2460,7 +2537,7 @@ def retrieve_finer_timeseries_data(request):
> >              res['reading'].append(ts_json)
> >
> >              count = count + 1
> > -            if count == 10:
> > +            if not start_time and not end_time and count == 10:
>
> document why you are stopping at 10?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/SEED-platform/seed/pull/863/files/07fcd5c43cfd1d50ca513fd8bd38d7410b999749#r57759601>
>